example/CMakeLists.txt

```diff
@@ -1,8 +1,22 @@
 #
-# Copyright (c) 2019 Vinnie Falco (vinnie.falco@gmail.com)
+# Copyright (c) 2021 Alan Freitas (alandefreitas@gmail.com)
 #
 # Distributed under the Boost Software License, Version 1.0. (See accompanying
 # file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 #
-# Official repository: https://github.com/vinniefalco/uri
+# Official repository: https://github.com/CppAlliance/url
+#
+
+source_group("" FILES
+    quicklook.cpp
+)
+
+#
+
+add_executable(quicklook
```

Comment:
This example helped me identify some bugs in the documentation examples. Even the first `parse_uri` example wasn't working before.